Airports/EGWU/Ownership.txt

```diff
@@ -2,11 +2,11 @@ SECTOR:Only WUAPP:0:0
 OWNER:WUA
 
 SECTOR:WUTWR:0:2000
-OWNER:WUT:WUA:LLN:LLS:LLF:TCNW:TCN:TC:LC:LNC:LWC:LSC:L
+OWNER:WUT:WUA:LLN:LLS:LLF:TCSE:TCS:TC:LD:LS:LSC:L
```

Comment:
Also need to change ownership of WUAPP in the Heathrow ownership file now (sorry!)

https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Airports/EGLL/Ownership.txt